### DISCUSSION TITLE
Does the day matter?

### DISCUSSION THREAD

**Discussion by stuartrainsworth** [dataplat/DataSaturdays#168] [Ideas]
One of the limitations of the SQLSaturday brand was that it contained "SQL" and "Saturday".  As we've seen over time, "SQL" no longer fits for the data platform; it's gotten increasingly broad.

Likewise, organizers occasionally tried to stretch the model to cover events on other days (Sundays or Fridays, for example).  Are you positioning this to be an event heled almost exclusively on Saturdays, or do yo want more flexibility than that?

DataDay, DataPlatfomDay....

**Comment by spaghettidba** [comment]
Well, many events are already held days other than Saturdays. I think the "Saturday" part is a sort of brand that has gained some success and maybe is worth keeping. Also, we own datasaturdays.com :)

Jokes aside, personally I'm very open to changing anything that makes sense changing. I'm not sure that "Saturday" is one of the things I'd change.

**Comment by Alex-Yates** [comment]
SQL Saturday was catchy because of illiteration on the S, but was restrictive to Saturdays.

Data Saturday is the worst of both worlds.

Data Day would have the nice illiteration, without the day of the week restriction, allowing organisers to choose a day that best suits their community.

I like "Data Day". Nice idea Stuart!

**Comment by Alex-Yates** [comment]
Or "Data Days". (Like DevOps Days).

Other ideas:
Data Dungeon.
Diverse Data.
Data Summit.
Data Dynamo.
Forum: Data.
Data Discord.
Talk: Data.
Data Congress.
DataConf.
Data Aggregate.
DataDump.
DBCC (Database Community Congress)

**Comment by stephlocke** [comment]
I personally think the saturday/weekend aspect is important- many people cannot get paid time off to attend training that is during the week. By actively prioritising weekend events it's more inclusive.

**Comment by Alex-Yates** [comment]
Is that not highly dependent on local culture? Many of the Friday events have found that Saturday's are less inclusive since in their culture orgs are likely to allow days off for training and running at the weekend tends to impact on family time. In their local culture this is considered less inclussive since it disproportionately effects families etc.

I guess I'm asking whether it's right for the central org to make assumptions about local cultures.

**Comment by gavincampbell** [comment]
My experience of these events suggests that you get a different demographic at the weekend than you do during the week, and there tends to be a drop-off as you get later into Saturday, q.v. the "Spartacus" effect at the raffle draw. 

Not that this is necessarily a reason to exclude one or the other, of course.

Also agree that this is culture-dependent, but having been to events in a few countries, I've never had the feeling that the overall registration numbers are greatly affected by what day the event is held.

**Comment by stuartrainsworth** [comment]
I agree that the day of the event draws different crowd; we did a similar low-cost event on 2 weekdays a couple of years ago, and most of the crowd had never been to a SQLSaturday (and remember that in Atlanta, we usually have 1000 registrations for SQLSaturday).

But Alex's question above gets to the heart of it: "whether it's right for the central org to make assumptions about local cultures."

I think if you're going to do this, then branding is important, and unless there's a specific driver to say "it must be on a Saturday", then permanently attaching it to that day isn't necessary.  So, let me rephrase this as a positive question rather than a negative?

What makes this event special?  Below are MY answers, and there are others that I'm not thinking of:

1. it's low-to-no cost (lunch + sponsors).
2. Speakers are volunteers, not paid
3. Its focused on the Microsoft Data Platform stack, but ancillary topics\technologies are encouraged.
4. Single day (plus precon options); This makes it easy for attendees\organizers\volunteers and avoids massive event sprawl
5 Builds\benefits the local user groups
6 A common brand across many events helps all events grow.

Given those priorities, I hope you can see why the specific day isn't important to me.

**Comment by datarelish** [comment]
Summary: Days and times should be left to the culture to decide. This is a big deal so I have put some information below for your reference.

Detail:
There will be particular dates that are specific and important to each religion. The Shap Calendar identifies the dates and purpose of each religious festival [https://www.interfaith.org.uk/resources/religious-festivals](https://www.interfaith.org.uk/resources/religious-festivals)

Why not call it '**SQLFamily Day**'? The hashtag is already widely adopted. Datafamily is the Tableau community's hashtag so that one isn't up for grabs easily, and it is well established. It allows for flexibility for people to decide on the day.

_Regarding the days_
Fridays and Saturday events can exclude practising Jews who follow Shabbath / Sabbath. The Sabbath begins at nightfall on Friday and lasts until nightfall on Saturday until the appearance of three stars on Saturday evening. The Sabbath starts a few minutes before sunset on Friday and runs until an hour after sunset on Saturday, lasting about 25 hours. In reality, some practising Jews will often try to get home at lunchtime on Friday to be absolutely sure that they are home in time. 
This is why we do not see many Jews at SQLBits, for example, because the free Saturday event excludes practising Jews by default, so they can only attend if they pay one of the paying days from Wednesday to Friday lunchtime. 
IIRC SQLSaturday was held on a Sunday in Israel. 
Seventh-Day Adventists also honour the Sabbath and keep the same time (nightfall to nightfall). 
Observant Christians will also keep the Sabbath but the commitment varies according to the interpretation.

_Regarding the Times_
Islam - Acknowledgement of Salawat prayer times.  Often, event breaks can be co-ordinated around the prayers times. Note that this will also include time for the ablution beforehand.
 
Catholic - The Angelus can be recited at 6am, noon and 6pm. If an event is held near one of the ancient Universities or Churches, the Angelus bells may ring out. This particular devotion is uncommon and the devotee can perform this prayer in private. 
 
Zoroastrian  - This is also known as Parsi faith. The Zoroastrians also pray five times a day. The prayer timings are less clear and more flexible. You may not come across many Zoroastrians since they number less than 200,000 throughout the world and their numbers dwindle at 10% per decade. 

Feel free to add others which I may have missed.

**Comment by Alex-Yates** [comment]
As Jen says, the Saturday is a sabbath day to many folks.

It's also a work day for many folks. If the objective is to promote running the event on a weekend day, we should recognise that in many countries Saturdays are not considered weekend days.

**Comment by HugoKornelis** [comment]
I personally believe that the "Saturday" part of the name is important. People associate it with the idea of a free event, with unpaid speakers, and usually on a day off. Austria had their SQLSaturday on Friday; Israel on Monday, and they still kept the name.
Other communities have copied the idea, and the "saturday" part of the name: Sat-R-Day and Sharepoint Saturday are two I know of.

I think "DataSaturdays" is a logical step. A lot of events have rebranded by replacing SQL with Data in their name.

**Comment by sqlworldwide** [comment]
Branding matters. If we can keep the name as is ( I am not ignoring some of the valid comments above) people will feel like SQL Saturday is alive.  As soon as we change it someone somewhere will use it and confuse.
It is a mindset. People will think SQL Saturday died with PASS.

**Comment by way0utwest** [comment]
I added an issue at #17 that is more general.

My US$0.02 here. In all the events I've been to and asked lots of people about this, I find that 
- about 30-40% would never be able to come to an event on a weekday because they work.
- about 30-40% would never come on a weekend. My wife thought this SQL Saturday was the dumbest thing ever and she'd never go.
- 20-30% will always come.

That being said, I think the day doesn't matter. I say we give guidance and let people decide if they want a DataSaturday on Tuesday or not. Other comments in #17

**Comment by steverezhener** [comment]
maybe we should just create a poll and see what name gets more votes?

**Comment by datarelish** [comment]
Agree with @steverezhener. We'd have to trust that people aren't playing the system with voter fraud but I hope sqlfamily are above that. To get better adoption, I think that the community should decide and this is transparent as well. It doesn't mean that people can't do their own thing if they want, of course. One issue with PASS was the inflexibility of their community event models so it was their branding or the high way, and we can learn from that experience.
